app/webpacker/components/CompetitionForm/FormSections/CompetitorLimit.js

```diff
@@ -9,10 +9,16 @@ export default function CompetitorLimit() {
   const {
     competitorLimit: {
       enabled: hasLimit,
-      autoAcceptEnabled,
+      autoAcceptPreference,
     },
   } = useFormObject();
 
+  const autoAcceptOptions = ['disabled', 'bulk', 'live'].map((status) => ({
```

Comment:
filter the options available to the delegate based on whether or not they're an admin